.github/ISSUE_TEMPLATE/new-content-needed.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New Content Needed
+about: Topic is missing and needs to be written
+title: "[CONTENT]"
+labels: content/missing-information
+assignees: ''
+
+---
+
+**What content needs to be created or modified?**
+A clear and concise description of what the problem is. Ex. There should be docs on how pub/sub works...
+
+**Describe the solution you'd like**
```

Comment:
Maybe also ask for a suggested section where the new content should be added?

Reply:
Good call. Added